packages/mwp-tracking-plugin/src/util/avro.js

```diff
@@ -23,14 +23,9 @@ const getPlatformAnalyticsLog = (
 		});
 		const publisher = pubsub.topic('analytics-log-json').publisher();
 		return (serializedRecord: string) => {
-			publisher.publish(new Buffer(serializedRecord)).then(
```

Comment:
also dropping this log, which wasn't actually very useful - just was a 'success' when an analytics log was sent. better to just log errors.